ansible_collections/arista/avd/molecule/eos_cli_config_gen_deprecated_vars/intended/configs/host1.cfg

```diff
@@ -0,0 +1,55 @@
+!
+interface Port-Channel2
+   switchport
+   switchport access vlan 100
+!
+interface Port-Channel3
+   switchport
+   switchport trunk allowed vlan 110
+   switchport trunk native vlan 10
+   switchport mode trunk
+   switchport trunk group group1
+   switchport trunk group group2
+!
+interface Port-Channel4
+   switchport
+   switchport trunk native vlan tag
+   switchport phone vlan 20
+   switchport phone trunk tagged
+   switchport mode trunk phone
+!
+interface Port-Channel5
+   switchport
+   switchport trunk private-vlan secondary
+   switchport pvlan mapping 2,3,4
+   switchport vlan translation in 23 50
+   switchport vlan translation out 25 49
+   switchport vlan translation 34 60
+!
+interface Ethernet1
+   switchport access vlan 100
+   switchport mode access
+   switchport
+!
```

Comment:
I looks weird that the ordering differs between Port-channel and Ethernet. Have you implemented all fixes consistently across both?

Reply:
Its the already existing code for deprecated vars